marklogic-unit-test-client/src/test/ml-modules/root/test/suites/Helpers/Delete All XML/test-delete-all-xml.xqy

```diff
@@ -0,0 +1,19 @@
+xquery version "1.0-ml";
+
+import module namespace test="http://marklogic.com/test/unit" at "/test/test-helper.xqy";
+import module namespace td="delete-all-xml-module" at "test-data/td.xqy";
+
+let $_ := test:delete-all-xml()
+
+(: check that it worked in separate transaction :)
+return 
+  xdmp:invoke-function(
```

Comment:
Instead of using xdmp:invoke(), you can use ; to force a second transaction.  You can perform your update in the first transaction, then read and assert on the update in the second transaction.

Reply:
True, I can change accordingly